run.sh

```diff
@@ -6,6 +6,8 @@ if [ "$1" == "cuda" ]; then
 fi
 
 set -e
+# This is required as new versions of alloy require rust 1.88 and default version is 1.86
```

Comment:
Comment needs to be updated (not required anymore)

But we might want to keep envvar in case revm version gets too new at some point